bar-trender-front-end/src/views/ListView.js

```diff
@@ -83,7 +83,8 @@ function ListView() {
         method: 'POST',
         body: JSON.stringify(filter),
         headers: {
-          'Content-Type': 'application/json'
+          'Content-Type': 'application/json',
+          'apiKey': '8dDc431125634ef43cD13c388e6eCf11'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7167715)
```suggestion
          "apiKey": '8dDc431125634ef43cD13c388e6eCf11'
```